src/server/implementation/cmd_parser.c

```diff
@@ -16,6 +16,7 @@ static const cmd_t player_commands[] = {
     {"Left", &move_left},
     {"Take", &take},
     {"Set", &set},
+    {"Connect_nbr", &connect_nbr},
```

Comment:
This file is outdated, please delete it. Add `connect_nbr` to `player_cmd_scheduler.c`.